source/common/http/headers.h

```diff
@@ -25,6 +25,7 @@ class HeaderValues {
   const LowerCaseString AccessControlMaxAge{"access-control-max-age"};
   const LowerCaseString AccessControlAllowCredentials{"access-control-allow-credentials"};
   const LowerCaseString Authorization{"authorization"};
+  const LowerCaseString B3{"b3"};
```

Comment:
Same here, please remove and define directly inside the Zipkin extension. Same for other X-B3 headers also.